docs/source/api/v1/parameters_profile_name.rst

```diff
@@ -27,6 +27,9 @@ Gets details about a specific :term:`Profile`'s :term:`Parameters`
 :Roles Required: None
 :Response Type:  Array
 
+.. deprecated:: ATCv4
```

Comment:
for consistency do you mind moving this up directly underneath the **** above GET? looks like its up there in the other deprecated endpoints and since this whole endpoint is deprecated